### PR TITLE
build: run native addon build in parallel

### PR DESCRIPTION
**Pull request by belochub**
Fixes: https://github.com/metarhia/jstp/issues/154

### CODE REVIEW

File: tools/build-native.js [hunk 1]
```diff
@@ -3,19 +3,22 @@
...
+const jobs = '-j' + os.cpus().length;
 
 fs.access('build', (error) => {
   if (error) {
     action = 'rebuild';
   }
 
-  const nodeGyp = childProcess.spawn('node-gyp', [action], { shell: true });
+  const nodeGyp = childProcess.spawn('node-gyp', [action, jobs],
+    { shell: true });
```
Comment (nechaido) [inline tools/build-native.js]: I'd prefer each argument to be on new line, like this:
```js
const nodeGyp = childProcess.spawn(
  'node-gyp',
  [action, jobs],
  {shell: true }
);
```

**Comment by belochub** [comment]
@lundibundi, I've added a possibility to use JOBS environment variable, PTAL.
About the second point: I don't think that there is any need to limit parallel jobs, most people even use job counts exceeding processor count.

File: tools/build-native.js (hunk 1 above)
Comment (aqrln) [inline tools/build-native.js]: Either that ↑ or move the last argument all the way right to the column where the first one starts.

Comment (aqrln) [inline tools/build-native.js]: For the clarity: I'd go for the latter option but I'm fine with it either way.

**Comment by lundibundi** [comment]
@belochub thanks, I wrote it as I personally tend to limit j flag to smth like that as I prefer to have at least one or 2 cpus free on my machine to allow seamless workflow even during system update/build/npm install process.

**Comment by aqrln** [comment]
Landed in 367fcd5e749b5e8b1791cc4c1a825ac5799352cc.

**Pull request closed without merging**